recipes/cppcheck/all/conanfile.py

```diff
@@ -44,7 +45,7 @@ def generate(self):
         tc.variables["HAVE_RULES"] = self.options.have_rules
         tc.variables["USE_MATCHCOMPILER"] = "Auto"
         tc.variables["ENABLE_OSS_FUZZ"] = False
-        tc.variables["FILESDIR"] = os.path.join(self.package_folder, "bin").replace('\\', '/')
+        tc.variables["FILESDIR"] = "bin"
```

Comment:
Why did you removed `os.path.join(self.package_folder, ` ?

Reply:
the installation path is manipulated with that environment variable. The .cfg files are now installed to the bin/cfg directory, the application will then find these files and the rmdir command is not nessessary anymore (because the folder doesn't exist)

Reply:
I approved but I'd like to have the previous author opinion about this change @ericLemanissier